understanding/20/keyboard.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Keyboard</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Everything can be done with a keyboard except freehand movements</dd>
-		<dt>Author task</dt><dd>Ensure pointer actions have a keyboard equivalent</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Everything can be done with a keyboard except freehand movements.</dd>
+         <dt>What to do</dt><dd>Ensure pointer actions have a keyboard equivalent.</dd>
+         <dt>Why it's important</dt><dd>Many people rely on the keyboard, including blind and some mobility impaired people.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>Many people rely on the keyboard or keyboard alternatives (speech, Morse code, scanning, etc.) , including blind and some mobility impaired people and need to be able to accomplish everything from the keyboard interface.</dd> 
```

Reply:
@GreggVan also suggested
```suggestion
         <dt>Why it's important</dt><dd>Many people rely on the keyboard (or alternate keyboards e.g. Morse code or Scanning or voice), including blind and some mobility impaired people.</dd> 
```

Reply:
As per prior comments, I believe this can be addressed briefly with a small change
```suggestion
         <dt>Why it's important</dt><dd>Many people rely on the keyboard interface, including blind and some mobility impaired people.</dd> 
```